### PR TITLE
feature | gds assessment | 466 | change user security

### PR DESCRIPTION
**Pull request by omarCreativeDev**
make change user security page gds compliant

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[Change user security page](https://trello.com/c/oQKlYQez/466-change-user-security-page)

File: src/app/features/change-user-security/change-user-security.component.html [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
+            {{ getFormErrorMessage('securityQuestionInput', 'maxlength') }}
           </span>
+        </span>
+        <input
+          class="govuk-input"
+          formControlName="securityQuestionInput"
+          id="securityQuestionInput"
+          name="securityQuestionInput"
+          [class.govuk-input--error]="submitted && getSecurityQuestionInput.errors"
+          placeholder=""
```
Comment (ChazUK) [inline src/app/features/change-user-security/change-user-security.component.html]: Can this be removed

File: src/app/features/change-user-security/change-user-security.component.html [hunk 2]
```diff
@@ -1,63 +1,75 @@
...
+            {{ getFormErrorMessage('securityAnswerInput', 'maxlength') }}
+          </span>
+        </span>
+        <input
+          class="govuk-input"
+          formControlName="securityAnswerInput"
+          id="securityAnswerInput"
+          name="securityAnswerInput"
+          [class.govuk-input--error]="submitted && getSecurityAnswerInput.errors"
+          placeholder=""
```
Comment (ChazUK) [inline src/app/features/change-user-security/change-user-security.component.html]: Can this be removed

File: src/app/features/change-user-security/change-user-security.component.ts [hunk 3]
```diff
@@ -5,87 +5,156 @@ import { Router } from '@angular/router';
...
-  public changeUserSecurityForm: FormGroup;
-  public displayError: boolean;
+  public form: FormGroup;
   public submitted: boolean;
-  public userDetails: {};
+  public userDetails: UserDetails;
 
   private username: string;
   private uid: string;
   private securityQuestion: string;
```
Comment (ChazUK) [inline src/app/features/change-user-security/change-user-security.component.ts]: Not really in the scope of this PR but do we need this variables? Are they being used anywhere else in the template, as their use in the component isn't really required

**Pull request merged**